testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -3578,19 +3578,17 @@ public void testCommentTable()
                     .skippingTypesCheck()
                     .containsAll("VALUES ('" + table.getName() + "', 'new comment')");
 
+            // comment deleted
+            assertUpdate("COMMENT ON TABLE " + table.getName() + " IS NULL");
+            assertThat(getTableComment(catalogName, schemaName, table.getName())).isEqualTo(null);
+
             // comment updated
             assertUpdate("COMMENT ON TABLE " + table.getName() + " IS 'updated comment'");
             assertThat(getTableComment(catalogName, schemaName, table.getName())).isEqualTo("updated comment");
```

Comment:
This block might be a little misleading now. "updated comment" was correct in the previous code, but it's a new comment now.

Reply:
I'm not saying about test coverage. I mean `null -> non-null` isn't "updated comment"  and it's a new comment in strict.

Reply:
I'll update the comment to something like `comment added to verify setting empty comment`? Or should I remove the assertion completely?

Reply:
@ebyhr I'll update the comment to something like `comment added to verify setting empty comment`? Or should I remove the assertion completely?

Reply:
updated the comment to be more true.